blockwise.go

```diff
@@ -341,7 +341,28 @@ func (b *blockWiseSession) WriteMsg(msg Message) error {
 	return b.WriteMsgWithContext(context.Background(), msg)
 }
 
+func (b *blockWiseSession) validateMessageSize(msg Message) error {
+	size, err := msg.ToBytesLength()
+	if err != nil {
+		return err
+	}
+	session, ok := b.networkSession.(*sessionTCP)
+	if !ok {
+		// Not supported for UDP session
+		return nil
+	}
+
+	if uint32(size) > session.peerMaxMessageSize {
```

Comment:
please extends condition about `session.srv.MaxMessageSize != 0 &&`

Reply:
I guess, here you mean to say `session.peerMaxMessageSize != 0  &&` , no ?

Reply:
Sure. 👍